src/helper/useLenis.ts

```diff
@@ -0,0 +1,18 @@
+'use client'
+import { useEffect } from 'react';
```

Comment:
I think it will be better if you combine both useLenis hook and Lenis wrapper into a single file.